.github/workflows/test-deploy.yml

```diff
@@ -122,22 +125,22 @@ jobs:
 
       - name: Install Node dependencies
         # We are checking for both root and theme node_modules folders to confirm the cache was found.
-        # The npm install deletes and recreates node_modules when needed,
+        # The pnpm install deletes and recreates node_modules when needed,
         # therefore we're skipping the command when the cache was hit.
         # Important: keep the detection logic in sync with the cache path above.
         run: |
           if [ -d "node_modules" ] && [ -n "$(find themes -maxdepth 2 -name 'node_modules' -type d 2>/dev/null)" ]; then
             echo "Node modules cache hit - skipping installation"
           else
             echo "Node modules cache miss - installing dependencies"
-            npm install --ignore-scripts --prefer-offline --no-audit
+            pnpm install --ignore-scripts --prefer-offline
```

Comment:
@mauteri I'm interested in learning more about using pnpm before it's used as a default on projects. My only concern about it is having certain tooling on some projects, but not on others. Does changing to pnpm offer advantages that all projects should use? Are there any downsides to using pnpm that need to be considered on projects?

Reply:
@aurooba could speak more to it, but there is apparently a huge performance benefit moving to `pnpm`. @landbryo is also moving to use it on Ziff Davis.